org.gnome.Cheese.yml

```diff
@@ -0,0 +1,75 @@
+id: org.gnome.Cheese
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+runtime-version: "3.34"
+command: cheese
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --device=all
+  - --metadata=X-DConf=migrate-path=/org/gnome/Cheese/
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+cleanup:
+  - /include
+  - /lib/girepository-1.0
+  - /lib/pkgconfig
+  - /man
+  - /share/aclocal
+  - /share/gir-1.0
+  - /share/pkgconfig
+  - /share/man
+  - /share/gtk-doc
+  - '*.la'
+  - '*.a'
+modules:
+  - name: gnome-desktop
+    buildsystem: meson
+    config-opts:
+      - -Ddebug_tools=false
+      - -Dudev=disabled
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gnome-desktop/3.34/gnome-desktop-3.34.2.tar.xz
+        sha256: 4081ba619733da33ec6ba98cd52170ee99a90439709df4df333e5c9e631f28ed
+  - shared-modules/lua5.3/lua-5.3.5.json
+  - name: libquvi-scripts
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/quvi/libquvi-scripts-0.9.20131130.tar.xz
+        sha512: 97de4cead30dc181da700cbf3a0ad66ee709fef17f449f75cd43e0376a4aa53c7b2174188130aaa34dcc7ddc578581111f3655cb10f4b0c92df2f75d32c72a02
+  - name: libquvi
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/quvi/libquvi-0.9.4.tar.xz
+        sha512: 287015e370e7b7f98f0f4c3717f5b044c8fcfb8b823c7988cd248e56b5fbbfa1a1aa421fbdafb38d727b79495267ccd4dead71587052117db53f29236268b651
+  - name: totem-pl-parser
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/totem-pl-parser/3.26/totem-pl-parser-3.26.4.tar.xz
+        sha256: c1576f551a67c2b3632a3dbea5d8b3e9966558af0cfeed58f07cec04728364f0
+  - name: totem-video-thumbnailer
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/totem-video-thumbnailer.git
+        commit: 78ddf6c113d439d8ad9ff1453cf433592fabe9fd
+  - name: gnome-video-effects
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gnome-video-effects/0.5/gnome-video-effects-0.5.0.tar.xz
+        sha256: 4dc68e9b38fdfc1e8e0414e2d7ee83ace78efdee76f30506cc9dcd07394ad0c8
+  - name: cheese
+    buildsystem: meson
+    config-opts:
+      - -Dintrospection=false
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/cheese/3.34/cheese-3.34.0.tar.xz
+        sha256: d1865600ac88012e136b1ec3db72a4634f57bc2035895277be792bb078627e73
+      - type: patch
+        path: org.gnome.Cheese.appdata.xml.in.patch.1
```

Comment:
Please remove the .1 suffix here and from the file too.

Reply:
Fixed in 34ae3aa thanks!

Reply:
Ok, now it's really fixed with 38affb1 late night fail.